js/modules/index.go

```diff
@@ -42,3 +44,15 @@ var Index = map[string]interface{}{
 	"k6/html":        html.New(),
 	"k6/ws":          ws.New(),
 }
+
+// PluginIndex holds a map of plugin modules to their respective implementations.
+var PluginIndex = map[string]interface{}{}
+
+// RegisterPluginModules takes care of registering a map of paths that a plugin exposes so they can be
+// loaded from the JavaScript VM.
+func RegisterPluginModules(modules map[string]interface{}) {
+	for path, impl := range modules {
+		importPath := fmt.Sprintf("k6-plugin/%s", path)
+		PluginIndex[importPath] = impl
+	}
```

Comment:
nitpick: I do think it will probably be easier to strip the prefix in the lookup ... although maybe the error message will need some work so it does actually include the "k6-plugin/"